plugins/modules/scaleway_database_backup.py

```diff
@@ -139,7 +139,7 @@
 RETURN = '''
 metadata:
     description: Backup metadata.
-    returned: when C(state=present), C(state=exported) or C(state=restored)
+    returned: when I(state=present), I(state=exported) or I(state=restored).
```

Comment:
I don't think we have a standard for `returned`, but since things like `success` are also OK, I'm not sure we should have a period here.

Reply:
There are several more places in this PR where you added periods in similar places.